.github/CONTRIBUTING.md

```diff
@@ -88,15 +88,15 @@ You can use issue labels to discover issues you could help out with:
 - [`bug` issues](https://github.com/Rocketseat/unform/labels/bug)
   are known bugs we'd like to fix;
 - [`enhancement` issues](https://github.com/Rocketseat/unform/labels/enhancement)
-  are features we're open to including.
+  are features we're open to include.
 
 The
 [`help wanted`](https://github.com/Rocketseat/unform/labels/help%20wanted)
 and
 [`good first issue`](https://github.com/Rocketseat/unform/labels/good%20first%20issue)
 labels are especially useful.
 
-You may find an issue is assigned. Please double-check before starting on this issue because somebody else is likely already working on it.
+You may find an issue that is assigned. Please double-check before starting on this issue because somebody else is likely already working on it.
```

Comment:
Reading your review, it got me thinking that we perhaps should change the way this is phrased. Maybe something like:

"When you see an issue that is already assigned,  please check to see if there isn't someone working on it already (maybe try asking in the issue). This is to prevent unnecessary work for everyone involved."

What do you guys think @lcnogueira @pellizzetti ?

Reply:
👍 I'm fine with that.

Reply:
It makes sense to me, @jpdemagalhaes. Actually, I hadn't realized that the old way was kind of repetitive. The way it's phrased now looks clearer :+1:

Reply:
Awesome!

@lcnogueira can you commit what we talked about so I can approve this PR?

Reply:
Done, @jpdemagalhaes :metal: .